src/test/java/edu/hawaii/its/api/service/TestGroupAttributeService.java

```diff
@@ -247,17 +247,17 @@ public void changeListServStatusTest() {
         String lastModTime4 = attributes.getWsAttributeAssigns()[0].getWsAttributeAssignValues()[0].getValueSystem();
         assertThat(lastModTime4, is(lastModTime3));
 
-        assertFalse(memberAttributeService.isOwner(GROUPING, username[1]));
+        // Invalid user can't change attribute status
         try {
-            groupAttributeService.changeGroupAttributeStatus(GROUPING, username[1], LISTSERV, true);
+            groupAttributeService.changeGroupAttributeStatus(GROUPING, "zzzzz", LISTSERV, true);
```

Comment:
Is "zzzzz" a possible uhUsername?

Reply:
Question: is 'zzzzzz' potentially a valid uhUuid (UH Number)? Unlikely, but, technically, yes. Place and underscore in the middle like this" 'zzz_zzz' and you have picked a username that is not in our namespace given our formatting rules.